src/Adapter/ORM/DoctrineORMAdapter.php

```diff
@@ -32,27 +32,27 @@ public function __construct(ManagerRegistry $registry)
         $this->registry = $registry;
     }
 
-    public function getNormalizedIdentifier($entity)
+    public function getNormalizedIdentifier($model)
     {
-        if (null === $entity) {
+        if (null === $model) {
             return null;
         }
 
-        if (!\is_object($entity)) {
+        if (!\is_object($model)) {
```

Comment:
We should move this check to a type declaration in the next major.

Reply:
Agree, this was only changed to make psalm/phpstan happy